example/tsconfig.json

```diff
@@ -13,13 +14,11 @@
     "noUnusedLocals": true,
     "outDir": "./dist",
     "removeComments": true,
+    "skipLibCheck": true,
     "sourceMap": true,
-    "strictNullChecks": true,
+    "strictNullChecks": false, // Prevents: JSX element type 'ReactElement<any> | null' is not a constructor function for JSX elements. Type 'null' is not assignable to type 'ElementClass'
```

Comment:
would be nice to keep `strictNullChecks` on... I don't really understand why this error is there in the first place tbh

Reply:
It is because of this: https://github.com/Microsoft/TypeScript/issues/11955

Reply:
That issue was supposedly fixed though(?)